saleor/payment/models.py

```diff
@@ -61,6 +61,14 @@ class Payment(models.Model):
     order = models.ForeignKey(
         "order.Order", null=True, related_name="payments", on_delete=models.PROTECT
     )
+    create_order = models.BooleanField(
```

Comment:
It seems to duplicate `complete_order` that I added earlier on?

Reply:
Removed